GitUI/UserControls/RevisionGrid/Columns/RevisionGraphColumnProvider.cs

```diff
@@ -216,7 +217,15 @@ void DrawItem(Graphics g, int index)
                     g.Clip = newClip;
                     g.Clear(Color.Transparent);
 
-                    DrawItem();
+                    try
+                    {
+                        DrawItem();
+                    }
+                    catch (Exception ex)
+                    {
+                        // consume since it does not bubble up to our handlers
+                        Debug.Write(ex);
+                    }
```

Comment:
try/catch is not free. this may regress scrolling perf. i don't know often this is called. can this be done at a higher level?

Reply:
Now, with one try/catch per graph row.